toolkits/search/arcade_search/tools/google_jobs.py

```diff
@@ -23,7 +23,7 @@ async def search_jobs(
     ] = None,
     language: Annotated[
         str,
-        "2-character language code to use in the Google Jobs search. ",
+        "2-character language code to use in the Google Jobs search. "
         f"E.g. 'en' for English. Defaults to '{DEFAULT_GOOGLE_JOBS_LANGUAGE}'.",
     ] = DEFAULT_GOOGLE_JOBS_LANGUAGE,
```

Comment:
For followup: What was this ultimately producing in the tool definition JSON? Could we catch this in the SDK and error gracefully?

Reply:
ah, this was making the annotation a tuple instead of a str, good catch!